`migrate_v3_to_v4.sql` doesn't create `timetable.migration` table

which is needed to operate. After the migration script run, the updated v4 `pg_timetable` produces the error:

```
2022-07-18 14:15:11.640 [INFO] Database connection established
2022-07-18 14:15:11.644 [ERROR] You need to upgrade your database before proceeding, use --upgrade option
2022-07-18 14:15:11.644 [INFO] Closing session
2022-07-18 14:15:11.646 [ERROR] [sql:WITH del_ch AS (DELETE FROM timetable.active_chain WHERE client_name = $1)
DELETE FROM timetable.active_session WHERE client_name = $1] [args:[loader]] [err:ERROR: relation "timetable.active_chain" does not exist (SQLSTATE 42P01)] [pid:22956] [time:1.1031ms] Exec
2022-07-18 14:15:11.646 [ERROR] [error:ERROR: relation "timetable.active_chain" does not exist (SQLSTATE 42P01)] Cannot finalize database session
exit status 3
```

--- a/extras/migrate_v3_to_v4.sql
+++ b/extras/migrate_v3_to_v4.sql
@@ -10,6 +10,17 @@ ALTER SCHEMA timetable RENAME TO timetable_v3;
 -- create the v4 initial schema
 CREATE SCHEMA timetable;
 
+CREATE TABLE timetable.migration(
+    id INT8 NOT NULL,
+    version TEXT NOT NULL,
+    PRIMARY KEY (id)
+);
+
+INSERT INTO
+    timetable.migration (id, version)
+VALUES
+    (0, '00259 Restart migrations for v4');
+
 CREATE TYPE timetable.command_kind AS ENUM ('SQL', 'PROGRAM', 'BUILTIN');
 
 CREATE TABLE timetable.task (

